include/RAJA/config.hxx.in

```diff
@@ -154,12 +154,21 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 //                 loop vectorization
 //
 
-#if defined(RAJA_COMPILER_ICC)
+#if defined (_WIN32)
+#define RAJA_RESTRICT __restrict
+#else
+#define RAJA_RESTRICT __restrict__
+#endif
+
+#if defined(RAJA_COMPILER_INTEL)
```

Comment:
This would be good as part of a larger change to ensure that `RAJA_COMPILER_XXXX` is always consistently set to `RAJA_COMPILER_${CMAKE_CXX_COMPILER_ID}`